src/components/TokenHistory.js

```diff
@@ -165,8 +273,22 @@ class TokenHistory extends React.Component {
       });
     }
 
+    const renderPage = () => {
+      const page = this.getPageNumber();
+      let span = null;
+      if (page === 1) {
+        span = <span>You are seeing transactions in real time</span>;
```

Comment:
Maybe use `receiving` instead of `seeing` and include a period at the end of the sentence.

Reply:
If you agree, then you need to update line 282 as well [1].

[1] https://github.com/HathorNetwork/hathor-wallet/pull/8/files#diff-ff7ed0cb94a737efdbdd9453182467c3R282

Reply:
I agree it's better, changed it!